doc/tools/python_signatures.py

```diff
@@ -0,0 +1,152 @@
+# TODO: clarify when there are several C++ signatures and only one Python one for the same function.
+#       i.e. calcHist() - http://docs.opencv.org/trunk/d6/dc7/group__imgproc__hist.html#ga4b2b5fd75503ff9e6844cc4dcdaed35d
+#       special case - http://docs.opencv.org/trunk/db/de0/group__core__utils.html#ga4910d7f86336cd4eff9dd05575667e41
+import codecs
+import re
+import sys
+import cv2
+
+from bs4 import BeautifulSoup
```

Comment:
surround this with a try catch. If it fails with an ImportError, print something and return right away.